### PR TITLE
Update misc tools

### PR DESCRIPTION
**Pull request by ericvn**
Hugo and Docker so far.

Add DNM to allow more changes within a single PR as other tool updates are found.

### CODE REVIEW

**Comment by jacob-delgado** [comment]
You beat me to it

BUF_VERSION=v1.32.0
HELM3_VERSION=v3.15.0
KUBECTL_VERSION=1.30.1
OC_VERSION=4.15.12
NODEJS_VERSION=20.13.1

**Review by jacob-delgado — APPROVED** [review]
Just update the others I mentioned. LGTM

File: docker/build-tools/Dockerfile [hunk 1]
```diff
@@ -54,7 +54,7 @@ ENV GO_BINDATA_VERSION=v3.1.2
 ENV GO_JUNIT_REPORT_VERSION=df0ed838addb0fa189c4d76ad4657f6007a5811c
 ENV HADOLINT_VERSION=v2.12.0
 ENV HELM3_VERSION=v3.14.4
-ENV HUGO_VERSION=0.125.7
+ENV HUGO_VERSION=0.127.1
```
Comment (jacob-delgado) [inline docker/build-tools/Dockerfile:57]: The latest is 126.1 not 127.1

**Comment by ericvn** [comment]
> You beat me to it
> 
> BUF_VERSION=v1.32.0 HELM3_VERSION=v3.15.0 KUBECTL_VERSION=1.30.1 OC_VERSION=4.15.12 NODEJS_VERSION=20.13.1

Updated

**Pull request merged**